src/App_material-overrides.scss

```diff
@@ -14,12 +14,18 @@
  * limitations under the License.
  */
 
-.App .mdc-button {
-  text-transform: none;
-}
+.App {
```

Comment:
I wouldn't be opposed to dropping these .App prefixes altogether

Reply:
Then you'd have to do something like `.mdc-button.mdc-button` to increase specificity, as discussed in the previous PR. I'd rather have the .App prefix.

Reply:
We could do `body .x` or we could ensure this file is always loaded later (slightly more risky).. 

... will leave at `.App` for now

Reply:
Late to the party, but please don't use `body .x` here, otherwise any more specific rules we write in the future will also need to include at least a tag, leading to weird rules like `body .my-component .mdc-button` or `.my-component div.mdc-button`. I'm happy to talk but I'd still strongly suggest that we keep `.App`.